### PR TITLE
possible typo

### PR DESCRIPTION
**Pull request by Jamesits**

### CODE REVIEW

**Comment by ryanhanwu** [comment]
I think there's a progressive relationship in this sentence, any other opinion?

**Comment by Jamesits** [comment]
The origin text is:

> If you did that, you'd (a) learn the answer, and (b) stop wasting my time.

I personally think the 2 points are in parallel, without any progressive or causal relationship.

File: README-zh_CN.md [hunk 1]
```diff
@@ -536,7 +536,7 @@ Jeff Bigler 的观察总结和这个相关也值得一读 (**[tact filters](http
 <a id="q4"></a>
 > 问题：我可以用 Bass-o-matic 文件转换工具将 AcmeCorp 档案转换为 TeX 格式吗？
 
-回答：试试看就知道了。如果你试过，你既知道了答案，就不用浪费我的时间了。
+回答：试试看就知道了。如果你试过，你既知道了答案，又不用浪费我的时间了。
```
Comment (fireairforce) [inline README-zh_CN.md]: 这里这样直译语句很不通顺吧

**Comment by ryanhanwu** [comment]
Close due to inactivity

**Pull request closed without merging**